src/cdk-experimental/config.bzl

```diff
@@ -6,6 +6,7 @@ CDK_EXPERIMENTAL_ENTRYPOINTS = [
     "menu",
     "listbox",
     "popover-edit",
+    "scrollable-table-body",
```

Comment:
I'm not sure whether this should be in its own separate module since it's very closely coupled to the table. Maybe we can have `material-experimental/table` that exports a `MatScrollableTableBody` module? I believe that would allow build tools to remove the code if it isn't used, while grouping under `table`.

Reply:
Yeah, I'd also prefer this. Alternatively a tertiary entry-point under `material(-experimental)/table/` could work too.